torch/_inductor/compile_fx.py

```diff
@@ -1023,7 +1027,9 @@ def bw_compiler(model: torch.fx.GraphModule, example_inputs):
     tracing_context = (
         torch._guards.TracingContext.get() or torch._guards.TracingContext(fake_mode)
     )
-    with V.set_fake_mode(fake_mode), torch._guards.tracing(tracing_context):
+    with V.set_fake_mode(fake_mode), torch._guards.tracing(
+        tracing_context
+    ), compiled_autograd.disable():
```

Comment:
It's not obvious to me how we are going to ensure we got all the spots this needs to be in.

Reply:
Should we make enabling python key tracing implicitly disable compiled_autograd?

Reply:
I'd be fine with this, although even if we did that I'm not sure it'd be exhaustive!